app/models/request/CreatePropertyPeriodicSubmissionRequest.scala

```diff
@@ -57,6 +57,13 @@ object CreatePropertyPeriodicSubmissionRequest {
         InternalError("No relevant entity found to convert from (to UpdatePropertyPeriodicSubmissionRequest)")
           .asLeft[CreatePropertyPeriodicSubmissionRequest]
     }
+
+    result.map(r =>
+      r.copy(ukOtherProperty = r.ukOtherProperty.flatMap(UkOtherProperty.convertToNoneIfAllFieldsNone(_)))
```

Comment:
this fixes only NONE, but as we noticed before when we tried to submit expenses, the request body was sent as empty 

Ref: `{"fromDate":"2023-04-06","toDate":"2024-04-05","ukOtherProperty":{}}` Pls debug this issue